README.md

```diff
@@ -16,7 +16,7 @@ This repo contains software for operating UUVs, developed by students at NTNU. T
 ## Documentation
 * TODO: Drivers and hardware specifics for each drone will be added to the wiki. Link them here.
 * TODO: How to adapt the software stack to new hardware.
-* A collection of master theses written by Vortex members:
+* A collection of master thesis written by Vortex members:
```

Comment:
Er ikke theses flertall av thesis da? Isåfall er theses riktig

Reply:
Theses er korrekt ja, virker som om codespell ikke helt skjønner kontekst

Reply:
fixed it just had to add a custom rule